### DISCUSSION TITLE
Is random erasing possible?

### DISCUSSION THREAD

**Discussion by kodohoi** [TorchIO-project/torchio#1054] [Q&A] [answered]
Hello.

I'm tackling the head CT analysis. However, my CT images sometimes lack some regions for several reasons, like artifacts.
Therefore, as a data augmentation, I would like to conceal some regions of normal images like Random Erasing (Z Zhong et al., 2017, arXiv).
Is it possible with Torchio?

Thanks in advance.

**Comment by fepegar** [comment, marked as answer]
Hi, @kodohoi. It is not implemented, but it is in principle possible to create such a transform. If you need it and would like to contribute, we will be happy to guide you.
